### ISSUE TITLE
exports map and .mjs extension

### ISSUE THREAD

**Issue by n1ru4l** [kamilkisiela/bob#16]
It seems like there is no mjs build being made and further, the exports map is completely stripped from the published package, despite being defined inside the repo package.json: https://github.com/n1ru4l/graphql-live-query/blob/933bd6d4a9d1b28ff5a090dc203632b9bf876bac/packages/in-memory-live-query-store/package.json#L37-L40 vs https://unpkg.com/browse/@n1ru4l/in-memory-live-query-store@0.6.0/package.json

Also, the published bundle does not contain a `.mjs` module file (only `.esm.js`) https://unpkg.com/browse/@n1ru4l/in-memory-live-query-store@0.6.0/ if not provided even in an esm (aka node 14) environment the commonjs version will be loaded.

TL;DR: The published package should contain an `index.mjs` file  and the package.json should have an `exports` map property (same asthe in-repo package.json).

**Comment by PabloSzx** [comment]
Done in Done in https://github.com/PabloSzx/bob-esbuild 👀
